src/components/Projects/WBS/WBSDetail/Task/Task.jsx

```diff
@@ -25,6 +25,7 @@ const Task = props => {
   const userPermissions = props.state.auth.user?.permissions?.frontPermissions;
   useEffect(() => {
     setIsCopied(false);
+    console.log('props.filteredTasks:', props.filteredTasks);
```

Comment:
Please, delete this debug output.

Reply:
@rauleffting thanks for the feedback! Fixed it, please review again.

Reply:
@rauleffting Hi, can you please check If your requested changes have been made, please approve this. 
Thanks.

Reply:
@one-community @rauleffting I deleted the console.log.